extensions/amp-accordion/amp-accordion.md

```diff
@@ -96,82 +98,222 @@ In this example, we display three sections, where the third section is expanded
+
+[example preview="inline" playground="true" imports="amp-accordion"]
+
+```html
+<amp-accordion id="myAccordion">
+  <section id="section1">
+    <h2>Section 1</h2>
+    <p>Bunch of awesome content</p>
+  </section>
+  <section>
+    <h2>Section 2</h2>
+    <div>Bunch of awesome content</div>
+  </section>
+  <section>
+    <h2>Section 3</h2>
+    <div>Bunch of awesome content</div>
+  </section>
+</amp-accordion>
+<button on="tap:myAccordion.toggle">Toggle All Sections</button>
+<button on="tap:myAccordion.toggle(section='section1')">
+  Toggle Section 1
+</button>
+```
+
+[/example]
 
-Set this attribute on the `<amp-accordion>` to opt out of preserving the collapsed/expanded state of the accordion.
+### `expand`
+
+The `expand` action expands the sections of the `amp-accordion`. If a section is already expanded, it will stay expanded. When called with no arguments, it will expand all sections of the accordion. Specify a specific section by adding the `section` argument and the corresponding `id` as the value.
+
+```html
+<button on="tap:myAccordion.expand">Expand All Sections</button>
+<button on="tap:myAccordion.expand(section='section1')">
+  Expand Section 1
+</button>
+```
+
+### `collapse`
+
+The `collapse` action collapses the sections of the `amp-accordion`. If a section is already collapsed, it will stay collapsed. When called with no argument, it will collapse all section of the accordion. Specify a specific section by adding the `section` argument and the corresponding `id` as the value.
+
+```html
+<button on="tap:myAccordion.collapse">Collapse All Sections</button>
+<button on="tap:myAccordion.collapse(section='section1')">
+  Collapse Section 1
+</button>
+```
 
 [/filter]<!-- formats="websites, stories" -->
 
-##### `expanded`
+## Events
+
+The `amp-accordion` events below trigger on accordion sections when clicked.
+
+[example preview="inline" playground="true" imports="amp-accordion"]
+
+```html
+<amp-accordion id="myAccordion">
+  <section id="section1" on="expand:myAccordion.expand(section='section2')">
+    <h2>Section 1</h2>
+    <p>Opening me will open Section 2</p>
+  </section>
+  <section id="section2" on="collapse:myAccordion.collapse(section='section1')">
+    <h2>Section 2</h2>
+    <div>Closing me will close Section 1</div>
+  </section>
+  <section>
+    <h2>Section 3</h2>
+    <div>Bunch of awesome content</div>
+  </section>
+</amp-accordion>
+```
 
-Set this attribute on a `<section>` to display the section as expanded on page load.
+[/example]
 
-##### `expand-single-section`
+### `expand`
 
-Set this attribute on the `<amp-accordion>` to only allow one `<section>` to be expanded at a time. If the user focuses on one `<section>` any other previously expanded `<section>` will be collapsed.
+The `expand` event triggers on the target `amp-accordion` section that changes from the collapsed state to the expanded state. Calling `expand` on an already expanded section will not trigger the event.
```

Comment:
Nit: `the event` or `an event`?

Reply:
the `expand` event. I specified. Thanks!